### PR TITLE
Add compression method

### LINKED ISSUE #137: Compressing Image
I searched through the issued but I couldn't find a way to just `compress` the image. 
I want to convert an image from any type (jpeg/png/webp) etc to any other, while compressing the size, keeping original dimensions
Can you point me to a direction

### PR DESCRIPTION
**Pull request by StckOverflw**
Add a simple compress method to the library based on the solution given in issue #137

If there is a better solution to this lmk, I'd be happy to implement that instead.

Closes #137

### CODE REVIEW

**Comment by silvia-odwyer** [comment]
@StckOverflw Hi Emma, thanks very much for the pull request, it's much appreciated! 😄 I'm just wondering if you could make the changes requested, I've marked where the fixes need to be made in the review above. After that, the test suite should pass ✅  

Thanks again, looking forward to merging the changes 👍

**Comment by StckOverflw** [comment]
I'm not seeing a review. Did you forget to submit it?

**Review by silvia-odwyer — CHANGES_REQUESTED** [review]

**Comment by silvia-odwyer** [comment]
@StckOverflw Hi Emma, It was marked pending, my apologies. The two changes I requested should be published now ✅

File: crate/src/transform.rs
Comment (silvia-odwyer) [inline crate/src/transform.rs:858]: This line needs a semi-colon at the end btw, the test suite should pass then

**Review by silvia-odwyer — CHANGES_REQUESTED** [review]

**Comment by StckOverflw** [comment]
I'm sorry, I didn't concentrate on what I was doing haha

**Comment by silvia-odwyer** [comment]
@StckOverflw No problem at all haha, and thanks again for your contribution! Everything looks good, so I'm going to merge this now ✅ 👍

**Review by silvia-odwyer — COMMENTED** [review]
Not sure if this PR was for Hactoberfest, but it should count as I've added the `hacktoberfest-accepted` label to this PR. I can't seem to approve the PR, not sure if that's needed for a PR to count towards Hacktoberfest. Anyway, just wanted to let you know about that just in case

**Comment by silvia-odwyer** [comment]
@StckOverflw Yeah I double-checked it, there's no way for me to approve the PR after merging ... so hopefully it should still count, but I can't be certain that being said

**Comment by StckOverflw** [comment]
Yes, it counted since the repo has the hacktoberfest label, thank you!

**Pull request merged**
